### PR TITLE
[v6r21] fix for bulk submission

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*TS
FIX: when a site was requested inside the job workflow description, and BulkSubmission was used, such site was not considered.

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
@atsareg  and @andresailer please look at the fix in https://github.com/DIRACGrid/DIRAC/pull/4022/commits/1264923eb9534f6f42bf57abcc5208f2fa9a12dd

File: WorkloadManagementSystem/scripts/dirac-admin-pilot-logging-info.py [hunk 1]
```diff
@@ -41,36 +40,24 @@ def setJobID(optVal):
...
 
 if uuid:
-  result = PilotsLoggingClient().getPilotsLogging(uuid)
+  result = WMSAdministratorClient().getPilotLoggingInfo(uuid)
   if not result['OK']:
     print 'ERROR: %s' % result['Message']
     DIRAC.exit(1)
-  printPilotsLogging(result['Value'])
-  DIRAC.exit(0)
+  print result['Value']
```
Comment (andresailer) [inline WorkloadManagementSystem/scripts/dirac-admin-pilot-logging-info.py]: No print statements please

File: WorkloadManagementSystem/scripts/dirac-admin-pilot-logging-info.py [hunk 2]
```diff
@@ -41,36 +40,24 @@ def setJobID(optVal):
...
     DIRAC.exit(1)
   for pilot in info['Value']:
-    logging = PilotsLoggingClient().getPilotsLogging(pilot['PilotJobReference'])
+    logging = WMSAdministratorClient().getPilotLoggingInfo(pilot)
     if not logging['OK']:
       print logging['Message']
-    printPilotsLogging(logging)
-  DIRAC.exit(0)
+      DIRAC.exit(1)
+    print logging['Value']
```
Comment (andresailer) [inline WorkloadManagementSystem/scripts/dirac-admin-pilot-logging-info.py]: No print statements please

File: WorkloadManagementSystem/scripts/dirac-wms-get-wn-parameters.py [hunk 3]
```diff
@@ -11,45 +11,49 @@
...
 from DIRAC.WorkloadManagementSystem.Utilities import JobParameters
 
-Script.setUsageMessage( '\n'.join( ['Get the parameters (Memory and Number of processors) of a worker node',
-                                    'Usage:',
-                                    '%s [option]... [cfgfile]' % Script.scriptName,
-                                    'Arguments:',
-                                    ' cfgfile: DIRAC Cfg with description of the configuration (optional)'] ) )
+Script.setUsageMessage('\n'.join(['Get the parameters (Memory and Number of processors) of a worker node',
+                                  'Usage:',
+                                  '%s [option]... [cfgfile]' % Script.scriptName,
```
Comment (andresailer) [inline WorkloadManagementSystem/scripts/dirac-wms-get-wn-parameters.py]:
```suggestion
                                  '  %s [option]... [cfgfile]' % Script.scriptName,
```

Comment (andresailer) [reply WorkloadManagementSystem/scripts/dirac-wms-get-wn-parameters.py]: This is needed to get the proper formatting in the documentation.

**Pull request merged**